web/src/pages/Route/service.ts

```diff
@@ -92,7 +92,7 @@ export const fetchUpstreamItem = (sid: string) => {
 export const checkHostWithSSL = (hosts: string[]) =>
   request('/check_ssl_exists', {
     method: 'POST',
-    data: hosts,
+    data: { hosts },
```

Comment:
Will it automatically convert parameters like {hosts:["xx.com"]} ?

Reply:
yes